packages/amplify-category-geo/src/service-utils/validateGeoJSONFile.ts

```diff
@@ -0,0 +1,82 @@
+import { readFileSync } from 'fs-extra';
+import Ajv from 'ajv';
+import { FeatureCollection, IdentifierOption } from './populateParams';
+import GeoJSONSchema from 'amplify-category-geo/schema/GeoJSONSchema.json';
+import { v4 as uuid } from 'uuid';
+import { printer } from 'amplify-prompts';
+
+
+export const validateGeoJSONFile = (geoJSONFilePath: string, uniqueIdentifier: string = 'id', identifierOption: IdentifierOption = IdentifierOption.RootLevelID) => {
```

Comment:
There is another limit of 1000 number of coordinates/vertices in the polygon https://docs.aws.amazon.com/location-geofences/latest/APIReference/API_BatchPutGeofenceRequestEntry.html#API_BatchPutGeofenceRequestEntry_Contents

Reply:
Added the vertices number check